File: asset/src/main/java/dev/exit/asset/asset/AssetItemRepository.java
package dev.exit.asset.asset;

import java.util.List;
import java.util.UUID;

import org.springframework.data.jpa.repository.JpaRepository;

import dev.exit.asset.asset.AssetItem;

public interface AssetItemRepository extends JpaRepository<AssetItem, UUID> {

	List<AssetItem> findByOwnerIdOrderByCreatedAtDesc(String ownerId);
}

File: asset/src/main/kotlin/dev/exit/asset/asset/AssetItemRepository.kt
package dev.exit.asset.asset

import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface AssetItemRepository : JpaRepository<AssetItem, UUID> {

    fun findByOwnerIdOrderByCreatedAtDesc(ownerId: String): List<AssetItem>
}
